pyproject.toml

```diff
@@ -5,4 +5,8 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[tool.mypy]
```

Comment:
to be properly usable for externals we need to be closer to strict than the defaults

i haven't yet take an look how much pain is in that change but i believe we need to disallow untyped defs

Reply:
91b5f1ad40d0d3e4d818ae9377c608cb766cbdbd